client/src/components/common/names-capture.vue

```diff
@@ -652,19 +655,18 @@ export default class NamesCapture extends Vue {
     this.validate(true)
   }
   validatePhrases (choice: string) {
-    if (!this.editMode) {
-      let name = this.nameChoices[choice]
-      if (this.entityPhraseRequired) {
-        let name = this.nameChoices[choice].toUpperCase()
-        if (this.entityPhraseChoices.every(phrase => {
-          phrase = phrase.replace(/[.*+\-?^${}()|[\]\\]/g, '\\$&')
-          return (name.search(new RegExp('(\\s)' + phrase + '(\\s|$)')) === -1)
-        })) {
-          this.messages[choice] = 'Your name must contain one of the required phrases'
-        }
-        if (this.entityPhraseChoices.some(phrase => name.startsWith(phrase))) {
-          this.messages[choice] = 'Your name must not begin with one of the required phrases'
-        }
+    if (this.entityPhraseRequired) {
+      const name = this.nameChoices[choice].toUpperCase()
+      if (this.entityPhraseChoices.every(phrase => {
+        phrase = phrase.replace(/[.*+\-?^${}()|[\]\\]/g, '\\$&')
+        return (name.search(new RegExp('(\\s)' + phrase + '(\\s|$)')) === -1)
+      })) {
+        this.messages[choice] = `A ${this.entityTypeText} name must include (but not start with)
```

Comment:
I updated the messages to include the entity type and entity phrases. This now applies to both edit and new modes.